README.md

```diff
@@ -4,14 +4,7 @@
 This is a starter application to help you get started on integrating the LINE Web Login feature into your website. Web Login lets users log in to your website using their LINE account.
 
 ## Getting started with LINE Login in Java with Heroku
-Here we'll go through how you can implement Web Login on a web server using Spring Boot and Heroku.
-
-### Requirements
-
- - JDK 1.8 or higher
- - Maven 3.0 or higher
- - Git
- - A free [Heroku](https://dashboard.heroku.com/) account
+Here we'll go through how you can implement Web Login on a web server using Spring Boot and Heroku. Before you start, get a free [Heroku](https://dashboard.heroku.com/) account.
```

Comment:
I think it is better that the Heroku account is a requirement. Rather than as a instruction.

As there are cases where the reader already has a Heroku account.